test/runtests.jl

```diff
@@ -42,11 +42,13 @@ missingvalue = -999.999
 err_number = 1.0e-9
 err_array = 0.0
 
-for c in map(name, Mimi.compdefs(m)), v in Mimi.variable_names(m, c)
+# for c in map(name, Mimi.compdefs(m)), v in Mimi.variable_names(m, c)
+for c in Mimi.compdefs(m), v in Mimi.variable_names(m, name(c))
 
-    #load data for comparison
-    filename = joinpath(@__DIR__, "../contrib/validation_data_v040/$c-$v.csv")
-    results = m[c, v]
+    # load data for comparison
+    orig_name = c.comp_id.comp_name
```

Comment:
I almost feel it would be cleaner to just rename the csv files to match the new comp name, rather than using this, which seems to go very much into the internals of things?